Calculator/Calculator/AbsoluteLayout.xaml

```diff
@@ -11,46 +12,48 @@
                        FontSize="Large" />
             </Frame>
             <AbsoluteLayout AbsoluteLayout.LayoutBounds="0, .18, 1, .2" 
                             AbsoluteLayout.LayoutFlags="All">
-                <Button Text="C" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="/" AbsoluteLayout.LayoutBounds=".33, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="X" AbsoluteLayout.LayoutBounds=".66, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="Back" AbsoluteLayout.LayoutBounds=".99, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="OperationHandler" Text="C" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="OperationHandler" Text="/" AbsoluteLayout.LayoutBounds=".33, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="OperationHandler" Text="X" AbsoluteLayout.LayoutBounds=".66, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="OperationHandler" Text="Back" AbsoluteLayout.LayoutBounds=".99, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
             </AbsoluteLayout>
             <AbsoluteLayout AbsoluteLayout.LayoutBounds="0, .3, 1, .2" 
                             AbsoluteLayout.LayoutFlags="All">
-                <Button Text="7" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="8" AbsoluteLayout.LayoutBounds=".33, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="9" AbsoluteLayout.LayoutBounds=".66, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="-" AbsoluteLayout.LayoutBounds=".99, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="NumberHandler" Text="7" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="NumberHandler" Text="8" AbsoluteLayout.LayoutBounds=".33, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="NumberHandler" Text="9" AbsoluteLayout.LayoutBounds=".66, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large"  Clicked="OperationHandler" Text="-" AbsoluteLayout.LayoutBounds=".99, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
```

Comment:
it's a good practice to name handlers kind of On<Something>Clicked,
so you could rename NumberHandler to OnNumberClicked for example and OperationHandler to some appropriate name as well